# Standard message for sharing vehicle informations

int16 id    # ID: vehicle identification number
float64[] x # X: vehicle state intended as system state and controller state
float64[] g # g: vehicle reference

